src/coordinates.rs

```diff
@@ -89,28 +89,24 @@ impl_from_integer!(u16);
 
 impl CoordinateNumber {
     pub fn gerber(&self, format: &CoordinateFormat) -> Result<String, GerberError> {
-        // Format invariants
-        if format.decimal > DECIMAL_PLACES_CHARS {
-            return Err(GerberError::CoordinateFormatError("Invalid precision: Too high!".into()))
+        fn rounded_div(dividend: i64, divisor: i64) -> i64 {
```

Comment:
We could ask in IRC or on Reddit whether something like that exists in the stdlib, or could possibly be added.

For now, you could create a `utils` module:

- Add a `utils.rs` file and put the function in there (with the `pub` modifier)
- Add a `mod utils` statement to `src/lib.rs`
- Add `use utils::rounded_div` to the top of this file